_posts/2025-03-13-CPANSec-Retrospective-2024.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title:  "CPANSec retrospective 2024"
+date:   2025-03-13 00:23:17 +0200
+categories: cpansec update
+author: Thibault Duponchelle
+excerpt: Here is the CPANSec 2024 Retrospective
+---
+
+That was a big year for CPANSec. 
+
+If it gets a bit under the radar to you, please find some of the achievements listed in this retrospective!
+
+## Common Vulnerabilities and Exposures (CVEs)
+This year was the year where CPANSec installed a new mindset for Perl 
+and their relationship towards Common Vulnerabilities and Exposures (CVEs).
+
+Until then, very few CVEs were emitted for Perl modules and tooling. 
+And people and authors were a bit reluctant to deal with that process.
```

Comment:
Suggestion: Either qualify this with "Our experience is that …", or add a reference or other source?

Also, it's usually best to avoid starting a sentence with "And" :sweat_smile: